README.md

```diff
@@ -15,17 +15,29 @@ $ yarn dev
 ```
 
 Setup database schema and data
+Uncomment the following line in docker-compose.yml
+```yml
+#    ports:
+#        - "5432:5432"
 ```
-$ npx hasura migrate apply
+
+Start the docker stack and load the sql dump
+```
+$ docker-compose up -d
+$ docker exec -i <container_name> psql --username postgres <postgres)password> < ./dump.sql
```

Comment:
There seems to be some confusion on the <postgres)password part. Shouldn't it just be this?
--username postgres <postgres)password> <

Reply:
Also, maybe we should clarify where the container name is to be gotten from

Reply:
We can put example in comment like
# e.g docker exec -i emo-spectre_postgres_1 psql --username postgres postgres < ./erd-dump.sql

Because it needs database name but not the password

Reply:
You are correct. I think with docker-compose the name will almost always be emo-spectre_postgres_1 so I added that to the readme. But I also added a note for if that does not work.

Pushpanjali is also correct that it needs just postgres for the database, not the password, so updated that as well.